src/main/java/de/retest/web/AttributesConfig.java

```diff
@@ -17,6 +19,13 @@
 	public static final String WIDTH = "width";
 	public static final String HEIGHT = "height";
 
+	public static final String TAG_NAME = "tagName";
+
+	public static final String TEXT = "text";
+
+	private static final HashSet<String> identifyingAttributeKeys = new HashSet<>( Arrays.asList( ABSOLUTE_HEIGHT,
```

Comment:
Left-hand side should be `Set`.

Reply:
Plus: it's somewhat odd to have two identifying attribute sets/lists. The problem is actually the underlying YAML, we should probably change.